apis/compute/v1beta1/computefirewallpolicyrule_types.go

```diff
@@ -0,0 +1,160 @@
+/*
+Copyright 2024.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	refs "github.com/GoogleCloudPlatform/k8s-config-connector/apis/refs/v1beta1"
+	commonv1alpha1 "github.com/GoogleCloudPlatform/k8s-config-connector/pkg/apis/common/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +kcc:proto=google.cloud.compute.v1.FirewallPolicyRuleMatcherLayer4Config
+type FirewallpolicyruleLayer4Configs struct {
```

Comment:
I think this type and the other one should be camel cased, right ?

Reply:
Since this is a Beta resource, we should keep the types the same(?) I copied from the existing one: pkg/clients/generated/apis/compute/v1beta1/computefirewallpolicyrule_types.go